bitnami/external-dns/templates/deployment.yaml

```diff
@@ -496,8 +496,12 @@ spec:
           ports:
             - name: http
               containerPort: {{ .Values.service.port }}
+          {{- if .Values.readinessProbe }}
```

Comment:
Could you add a variable enabled and change the line to `{{- if .Values.readinessProbe.enabled }}`?

Reply:
done, thanks @Mauraza